src/haddock/libs/libplots.py

```diff
@@ -83,7 +83,188 @@ def read_capri_table(capri_filename, comment="#"):
     return capri_df
 
 
-def box_plots(capri_filename, cl_ranking, png, dpi):
+def in_capri(column, df_columns):
+    """
+    Check if the selected column is in the set of available columns.
+
+    Parameters
+    ----------
+    column : str
+        column name
+    df_columns : pandas.DataFrame.columns
+        columns of a pandas.DataFrame
+    Returns
+    -------
+    resp : bool
+        if True, the column is present
+    """
+    resp = True
+    if column not in df_columns:
+        log.warning(f"quantity {column} not present in capri table")
+        resp = False
+    return resp
+
+
+def update_layout_plotly(fig, x_label, y_label, title=None):
+    """
+    Update layout of plotly plot.
+
+    Parameters
+    ----------
+    fig : plotly Figure
+        figure
+    x_label : str
+        x axis name
+    y_label : str
+        y axis name
+    title : str or None
+        plot title
+    """
+    px_dict = {
+        "title": title,
+        "xaxis": dict(
+            title=x_label,
+            tickfont_size=14,
+            titlefont_size=40,
+            ),
+        "yaxis": dict(
+            title=y_label,
+            tickfont_size=14,
+            titlefont_size=40,
+            ),
+        "legend": dict(x=1.01, y=1.0, font_family="Helvetica", font_size=16),
+        "hoverlabel": dict(font_size=16, font_family="Helvetica")
+        }
+    fig.update_layout(px_dict)
+    return
+
+
+def update_layout_matplotlib(x_label, y_label):
+    """
+    Update layout of matplotlib plot.
+
+    Parameters
+    ----------
+    x_label : str
+        x axis name
+    y_label : str
+        y axis name
+    """
+    plt.title(f"{x_label} vs {y_label}", fontsize=40)
```

Comment:
this `plt` is defined outside the function scope, you could pass it as an argument and also return it

```python
def update_layout_matplotlib(_plt, x_label, y_label):
    # do things
    return _plt

# somewhere else in the code
updated_plt = update_layout_matplotlib(plt, TITLE_NAMES[x_ax], TITLE_NAMES[y_ax])
```

something like this

Reply:
I don't know..since both `plt` and `px` are aliases to imported libraries, does it make sense to pass them to the functions?
From what I've seen they're always called directly from within functions to modify the current plot..